images/linux/ubuntu2004.json

```diff
@@ -0,0 +1,324 @@
+{
+    "variables": {
+        "commit_url": "{{env `COMMIT_URL`}}",
+        "client_id": "{{env `ARM_CLIENT_ID`}}",
+        "client_secret": "{{env `ARM_CLIENT_SECRET`}}",
+        "subscription_id": "{{env `ARM_SUBSCRIPTION_ID`}}",
+        "tenant_id": "{{env `ARM_TENANT_ID`}}",
+        "resource_group": "{{env `ARM_RESOURCE_GROUP`}}",
+        "storage_account": "{{env `ARM_STORAGE_ACCOUNT`}}",
+        "temp_resource_group_name": "{{env `TEMP_RESOURCE_GROUP_NAME`}}",
+        "location": "{{env `ARM_RESOURCE_LOCATION`}}",
+        "virtual_network_name": "{{env `VNET_NAME`}}",
+        "virtual_network_resource_group_name": "{{env `VNET_RESOURCE_GROUP`}}",
+        "virtual_network_subnet_name": "{{env `VNET_SUBNET`}}",
+        "private_virtual_network_with_public_ip": "{{env `PRIVATE_VIRTUAL_NETWORK_WITH_PUBLIC_IP`}}",
+        "image_folder": "/imagegeneration",
+        "commit_file": "/imagegeneration/commit.txt",
+        "imagedata_file": "/imagegeneration/imagedata.json",
+        "metadata_file": "/imagegeneration/metadatafile",
+        "installer_script_folder": "/imagegeneration/installers",
+        "helper_script_folder": "/imagegeneration/helpers",
+        "vm_size": "Standard_DS2_v2",
+        "capture_name_prefix": "packer",
+        "image_version": "dev",
+        "image_os": "ubuntu20",
+        "github_feed_token": null,
+        "go_default": "1.14",
+        "go_versions": "1.12 1.13 1.14"
```

Comment:
May be it makes sense to keep only Go 1.14 for now? Any other version can be easily installed via `setup-go`